g11n-ws/vip-manager-i18n/src/main/resources/application.properties

```diff
@@ -4,6 +4,10 @@
 ##
 
 #build version information
+build.name=vip-i18n-manager
+build.author=VMware
+build.version=
```

Comment:
The value of build.version should be overwritten in the build.gradle and the value should come from the cmd parameter of build creation, right? But i don't see how this work.

Reply:
Have synced this with @junw-33455432 , he will overwrite the version value when publish build, as the same with what he did for the existing builddate&buildnumber&branch, so I needn't write such task.